src/main/java/com/github/maven_nar/Msvc.java

```diff
@@ -132,6 +132,12 @@ public void configureCCTask(final CCTask task) throws MojoExecutionException {
       envVariable.setKey("TMP");
       envVariable.setValue(getTempPath());
       task.addEnv(envVariable);
+      final String envInclude = System.getenv("INCLUDE");
```

Comment:
According to Microsoft documentation https://docs.microsoft.com/en-us/cpp/build/reference/cl-environment-variables CL compiler uses the environment variable `INCLUDE`, and if it's set incorrectly that already adds uncertainty.  So, it may be not the plugin's responsibility to work around messy environment settings.  It might be redundant to add paths from `INCLUDE` variable on the command line as `-I` options, except that `cl` as run by maven is executed in an environment which does not contain `INCLUDE` in it.